test-common/src/test/scala/org/scalajs/testcommon/SerializerTest.scala

```diff
@@ -12,6 +12,8 @@
 
 package org.scalajs.testcommon
 
+import java.io._
```

Comment:
Unused import?

Reply:
Removed unused import.

And squashed commits into https://github.com/scala-js/scala-js/pull/3631/commits/1aaf291a4fe323303a69c15e55437e0b0ef3a264